test/transport/custom_comm/CustomComm.cpp

```diff
@@ -0,0 +1,179 @@
+#include "CustomComm.hpp"
+
+#include <stdint.h>
+
+CustomComm::CustomComm() : buffer(2000)
+{
+    max_payload = 20;
+    uxr_set_custom_transport_callbacks(&master_, true, open, close, write, read);
+    uxr_init_custom_transport(&master_, this);
+
+    uxr_set_custom_transport_callbacks(&slave_, true, open, close, write, read);
+    uxr_init_custom_transport(&slave_, this);
+}
+
+CustomComm::~CustomComm()
+{
+    uxr_close_custom_transport(&master_);
+    uxr_close_custom_transport(&slave_);
+}
+
+bool CustomComm::open(uxrCustomTransport* transport)
+{   
+    (void) transport;
+    return true;
+}   
```

Comment:
I think it is more elegant like this, however this is just my opinion
```suggestion
bool CustomComm::open(uxrCustomTransport* /*transport*/)
{   
    return true;
}   
```